_topic_maps/_topic_map.yml

```diff
@@ -19,3 +19,10 @@ Distros: openshift-service-mesh
 Topics:
 - Name: Updating OpenShift Service Mesh
   File: ossm-updating-openshift-service-mesh
+---
+Name: Observability
+Dir: observability
+Distros: openshift-service-mesh
+Topics:
+- Name: Configuring OpenShift Monitoring
+  File: ossm-config-openshift-monitoring
```

Comment:
This sidebar link doesn't seem to work. Getting 404 when I click on the "Observability" section.

Reply:
It won't work yet, and this is also likely to change before TP 1.